frontend/src/components/posts/Post.jsx

```diff
@@ -0,0 +1,34 @@
+//react core
```

Comment:
exactly same comment here. I think we should try to be more semantic when we create files, functions and return elements from those functions. Honestly, I still don't understand what these three files Comments.jsx, FoodPost.jsx and Post.jsx are doing...